### PR TITLE
Add rhythm indicators on Controller

### PR DESCRIPTION
**Pull request by judy-n**
- Made a fix so the "Purr-fect", "Nice", "Oops" text shows on controller

### CODE REVIEW

File: Assets/Scripts/PlayerScripts/PlayerAction.cs [hunk 1]
```diff
@@ -2,43 +2,51 @@
 using System;
 using System.Collections;
 using System.Collections.Generic;
+using Unity.VisualScripting;
```
Comment (jamilfaisal) [inline Assets/Scripts/PlayerScripts/PlayerAction.cs]: I don't think this import is needed

File: Assets/Scripts/PlayerScripts/PlayerAction.cs [hunk 2]
```diff
@@ -78,13 +86,41 @@ private void Hit()
...
+
+    public void TriggerScoreCalculation(InputAction.CallbackContext context)
+    {
+        if (context.performed)
+        {
+            if (Time.time > 5)
+            {
+                if (_inputIndex < timeStamps.Count)
+                {
+                    if (Math.Abs(_audioTime - (_timeStamp)) < _marginOfError)
```
Comment (jamilfaisal) [inline Assets/Scripts/PlayerScripts/PlayerAction.cs]: I think lines 108 - 119 are the same as lines 58 - 68. If that's the case, can you please refactor these lines into one function and call in both places

**Review by jamilfaisal — CHANGES_REQUESTED** [review]
Works on both my xbox and switch controllers. Nice work! Will approve after my two comments are addressed

File: Assets/Scripts/PlayerScripts/PlayerAction.cs (hunk 1 above)
Comment (judy-n) [reply Assets/Scripts/PlayerScripts/PlayerAction.cs]: Fixed!

File: Assets/Scripts/PlayerScripts/PlayerAction.cs (hunk 2 above)
Comment (judy-n) [reply Assets/Scripts/PlayerScripts/PlayerAction.cs]: Fixed this

**Review by jamilfaisal — APPROVED** [review]

**Review by lilvedaes — APPROVED** [review]
LGTM c:

**Pull request merged**